daterangeparser/parse_date_range.py

```diff
@@ -240,7 +235,12 @@ def parse(text):
   - All times are ignored, assuming they are placed either before or after each date, otherwise they will cause an error.
   - The separators that are allows as part of the date range are `to`, `until`, `-`, `--` and `->`, plus the unicode em and en dashes.
   - Other punctuation, such as commas, is ignored.
-  
+
+  :param text: The string to parse
+  :param allow_implicit: If implicit dates are allowed
```

Comment:
It'd be great if you could give an example here of an 'implicit date' to help out any users that are less familiar with the term.